### ISSUE TITLE
Pseudo-element colours are no longer overridden by Windows High Contrast in Chrome/Edge 124

### ISSUE THREAD

**Issue by querkmachine** [alphagov/govuk-frontend#4962]
## Description of the issue

Versions of Google Chrome and Microsoft Edge from 124 onwards do not apply the expected override colours to pseudo-elements in Windows High Contrast Mode.

Significant issues:

- Lists — bullet points and number markers are always black instead of matching the text colour
- Checkboxes — checkbox border is always rendered as black instead of matching the text colour; the check image is also always rendered in black
- Radios — radio button border is always rendered as black instead of matching the text colour; the radio's checked state is also always rendered in black

Less significant issues:

- Accordion — toggle arrows are always black instead of matching the link colour
- Details — toggle arrow is always blue instead of matching the link colour
- Back link — back arrow is always blue instead of matching the link colour
- Breadcrumbs - separator arrows are always black instead of matching the text colour
- Header — the mobile menu toggle's arrow is always rendered as black instead of matching the text colour

### Screenshots

Expected result in Edge 123 on Windows 10 (via AssistivLabs)

![White on black screenshot of a simple question with three checkboxes below it. The first checkbox is checked.](https://github.com/alphagov/govuk-frontend/assets/1253214/908ce546-5ef9-4ff2-8da8-cc8130f4c32a)

Unexpected result in Edge 124 on Windows 10 (via AssistivLabs)

![White on black screenshot of a simple question with what should be three checkboxes below it, however the checkboxes aren't visible.](https://github.com/alphagov/govuk-frontend/assets/1253214/bcd20b94-f529-4446-8086-d442dc863c91)

## Steps to reproduce the issue

Activate Windows High Contrast Mode on a Windows 10 or Windows 11 PC. View the design system examples or review app examples using Google Chrome 124 or Edge 124 or newer.

## Actual vs expected behaviour

Elements should be rendered in a colour appropriate to the High Contrast Mode being used, rather than being static.

## Environment

Reproduced in:

- Operating system: Windows 10, Windows 11
- Browser: Chrome 124, Edge 124
- GOV.UK Frontend Version: 5.3.1

I also tested in Edge 123 on Windows 10 and Firefox 125 on Windows 11, neither of which exhibited this issue.
